tables/it-it-comp6.utb

```diff
@@ -1,9 +1,23 @@
-# liblouis: Italiano
-#
-#  Copyright (C) 2004-2008 ViewPlus Technologies, Inc. www.viewplus.com
-#  Copyright (C) 2004-2006 JJB Software, Inc. www.jjb-software.com
-#  Copyright (C) 2012 Mesar Hameed <mesar.hameed@gmail.com>
-#
+# liblouis: Italian 6 dot Braille table
+
+# Copyright (C) 2004-2008 ViewPlus Technologies, Inc. www.viewplus.com
+# Copyright (C) 2004-2006 JJB Software, Inc. www.jjb-software.com
+# Copyright (C) 2012 Mesar Hameed <mesar.hameed@gmail.com>
+# Copyright (C) 2025 Tommaso Nonis - Vision Dept SRL <tommaso@visiondept.it>
```

Comment:
It looks like Tommaso did not start from it-it-comp6.utb, but from fr-bfu-comp6.utb. In other words, all the copyright lines from 2004-2012 are not relevant anymore.


would like to